spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -252,6 +269,23 @@ def get_app_store_versions(filter: {}, includes: Spaceship::ConnectAPI::AppStore
         return resps.flat_map(&:to_models)
       end
 
+      #
+      # B2B
+      #
+
+      def disable_b2b
+        update(attributes: {
+          distributionType: DistributionType::APP_STORE
+        })
```

Comment:
I am getting an error on this because the educational discount type is still `NOT_APPLICABLE`, but that is not acceptable when distribution type is 'APP_STORE'

`When an app has NOT_APPLICABLE education discount, only CUSTOM app distribution is possible`

Reply:
Ahhhhh.... good catch! I will make that change 😊

Reply:
This should be fixed now!